app/docs/form-engine-client-setup/sections/quickbase.section.html

```diff
@@ -0,0 +1,27 @@
+<doc-section>
+  <doc-header id="quickbase">Quickbase</doc-header>
+  <doc-caption>Actions required to be performed on quickbase</doc-caption>
+
+  <h5>Copy Application</h5>
+    <ul>
+      <li>
+        Go to Settings under the application name
+      </li>
+      <li>Go to App Management</li>
+      <li>Copy app</li>
+        <ul>
+          <li>Data: select copy this application WITHOUT data</li>
+          <li>Users & Roles: Copy users in their roles</li>
```

Comment:
As such we'll need instructions for how to do that (create and maintain the QuickStart role).